src/build/leo/aleo-sdk/create-aleo-app/installation.md

```diff
@@ -1 +1,32 @@
+# Create Aleo App
 # Installation
+
+<a href="https://www.npmjs.com/package/create-aleo-app"> <img alt="Create Aleo App" src="https://img.shields.io/npm/l/create-aleo-app?label=NPM%20-%20Create-Aleo-App&labelColor=green&color=blue" /></a>
+
+## Scaffolding Your First Aleo Project
+
+:::note
+**Compatibility:**
+[Node.js](https://nodejs.org/en/) version 18+
+:::
+
+With NPM:
+
+```bash
+npm create aleo-app@latest
+```
+
+1. Enter the project name.
+2. This will prompt you to choose a preferred framework from the following options:
+   - `React`
+   - `Node.js`
+   - `Vanilla (JavaScript)`
+
+3. If you choose `React` as your framework, the supported templates are:
+   - `JavaScript` + `Leo`
+   - `TypeScript` + `Leo`
+   - `TypeScript` + `Next.js`
+     
+## More Information
+
+Based off of create-vite: https://github.com/vitejs/vite/tree/main/packages/create-vite
```

Comment:
Should this read "Based on `create-vite`" ??

Reply:
This is from the current documentation. The best example is on Vite. I think that is the reason they highlighted `vite` at the end, but we also have the rest. Do you think it should remain as it is, or should I add the other frameworks too?

Reply:
You are right, I converted it to "Based on create-vite"